packages/jsii-rosetta/bin/jsii-rosetta.ts

```diff
@@ -166,6 +168,55 @@ function main() {
         }
       }),
     )
+    .command(
+      'transliterate [ASSEMBLY..] [LANGUAGE..]',
+      '(EXPERIMENTAL) Transliterates the designated assemblies to the specified language (or all supported languages)',
+      (command) =>
+        command
+          .positional('ASSEMBLY', {
+            type: 'string',
+            string: true,
+            default: new Array<string>(),
+            required: true,
+            describe: 'Assembly to transliterate',
+          })
+          .positional('LANGUAGE', {
+            type: 'string',
+            string: true,
+            default: new Array<string>(),
+            describe: 'Language ID to transliterate to',
+          })
+          .option('tablet', {
```

Comment:
What's that? Not sure I understand this option (and `strict` as well). Maybe add some `.example()`s?